tests/unit/test_charts.py

```diff
@@ -0,0 +1,3 @@
+def test_charts_route_returns_ok(client):
```

Comment:
You should be able to test some of the resulting html for the empty case pretty easily. Can you write a quick test for that?

And maybe just throw one record in the DB and test for that?